src/nvim/api/vim.c

```diff
@@ -1268,6 +1269,67 @@ Dictionary nvim_get_color_map(void)
   return colors;
 }
 
+/// Returns a Context map for the current editor state.
+///
+/// @param  types  Context types ("regs", "jumps", "buflist", "gvars", ...)
+///                to gather, or [|v:null|] for all (see |context-types|).
```

Comment:
mentioning `v:null` in API docstrings is going to be confusing, because these docstrings must apply to all clients, not just the VimL client. NIL is always mentioned instead.